templates/Default/engine/Default/game_join.php

```diff
@@ -151,17 +151,12 @@
 			<td>
 				<table class="center">
 					<tr>
-						<td colspan="3">Trading</td>
-					</tr>
-					<tr>
-						<td>Combat<br />Strength</td>
 						<td>
-							<img width="440" height="440" id="graphframe" src="images/race/graph/graph<?php echo $SelectedRaceID; ?>.gif" alt="Race overview" />
+							<div style="width: 550px; height: 550px;background: noone;" id="graphframe"></div>
+							<script>
+								graph_by_plotly('<?php echo $SelectedRaceID?>');
```

Comment:
JavaScript can't be included mid-HTML anymore. To see why, open this page in Chrome, open the inspector and turn off caching (to exaggerate the effect), and then reload the page over and over again. What you'll see is that the page rendering is interrupted while the JS is being run, and since this occurs in the middle of a table, it momentarily breaks the page formatting.

See d0ec278efebc8ad845eb15e25fc74ad4dbfe6741 for how I solved this problem with our list.js implementation. Unfortunately, that doesn't support completely generic JS execution, so I'm not sure what the best way to solve this problem is. (Do we create another if-clause specific to running this one function? That's going to get pretty out of control if we continue to add more JS, but we need to get it into `EndingJavascript.inc` somehow.)

Reply:
It's not just this script tag, it's also the placement of `<script src='https://cdn.plot.ly/plotly-latest.min.js'></script>`. Both of these needs to go inside `EndingJavascript.inc`. Take out both and you'll see that the page renders properly.